test/system/inspect.test.js

```diff
@@ -653,12 +653,11 @@ test('Pipfile package found conditionally based on python version', (t) => {
     .then(() => {
       return plugin.inspect('.', 'Pipfile');
     })
-    .then((result) => {
-      const pkg = result.package;
-      t.notOk(pkg.dependencies.black, 'black dep ignored');
-      t.notOk(pkg.dependencies.stdeb, 'stdeb dep ignored');
-
-      t.end();
+    .catch((error) => {
+      t.match(
+        normalize(error.message),
+        'No dependencies detected in manifest.'
+      );
```

Comment:
We are no longer returning an empty dependencyTree and asserting that the object is empty but returning a real error msg.